examples/llama.py

```diff
@@ -13,7 +13,9 @@
 from tinygrad.tensor import Tensor
 from tinygrad.nn.state import safe_load, torch_load, load_state_dict, get_parameters
 from extra.models.llama import Transformer, convert_from_huggingface
-from sentencepiece import SentencePieceProcessor
+from sentencepiece import SentencePieceProcessor, sentencepiece_model_pb2
+import gguf
```

Comment:
I don't want to use this (certainly not required to run llama.py). Can we open this file format in state?